README.md

```diff
@@ -52,7 +52,7 @@ the Travis build before the build will go green.
 ## Running locally
 
 ./run.sh: Simulate Travis in determining whether someone has won.
-./run.sh <PR>: Simulate Travis in determing whether a PR can merge.
+./run.sh \<PR>: Simulate Travis in determing whether a PR can merge.
```

Comment:
Can you also fix the line breaks?  Right now it runs these two lines together.

Reply:
I also changed it to be backticked, which I think I like better. But I'll change it back if people disagree.